lib/server/db/seeds/seed.js

```diff
@@ -0,0 +1,31 @@
+var mongoose = require("../connect")();
+var seedConnections = require("./connection");
+var seedUsers = require("./user");
+
+seedUsers([
+    {
+        firstName: "mike",
+        lastName: "jones",
+        username: "mike",
+        password: "mike"
+    },
+    {
+        firstName: "josh",
+        lastName: "cox",
+        username: "josh",
+        password: "josh"
+
+    }
+])
+    .then(function() {
```

Comment:
Since our seed modules are functions that return `Promise`s, we can chain different seed configs together synchronously. In this particular case, we want this since we can't add connections until our users have been added to the database.